### PR TITLE
ref(feedback): Refactor feedback layout and some analytics

### PR DESCRIPTION
**Pull request by ryan953**
There's a new tooltip on the Feedback title:
SCR-20231205-oaxs

### CODE REVIEW

File: static/app/components/feedback/useHaveSelectedProjectsSetupFeedback.tsx [hunk 1]
```diff
@@ -22,16 +21,7 @@ function getSelectedProjectList(
   return selectedProjects.map(id => projectsByProjectId[id]).filter(Boolean);
 }
 
-export function useHasOrganizationSetupFeedback() {
```
Comment (ryan953) [inline static/app/components/feedback/useHaveSelectedProjectsSetupFeedback.tsx:25]: this wasn't used

File: static/app/components/feedback/useHaveSelectedProjectsSetupFeedback.tsx [hunk 2]
```diff
@@ -46,10 +36,3 @@ export function useHaveSelectedProjectsSetupFeedback() {
     fetching,
   };
 }
-
-export function useFeedbackHasSlug() {
```
Comment (ryan953) [inline static/app/components/feedback/useHaveSelectedProjectsSetupFeedback.tsx:50]: replaced with a call to `useCurrentFeedbackId()` and then a boolean check on the result.

File: static/app/utils/analytics/feedbackAnalyticsEvents.tsx [hunk 3]
```diff
@@ -2,17 +2,13 @@ export type FeedbackEventParameters = {
...
-  'feedback.index-setup-button-clicked': {};
   'feedback.index-setup-viewed': {};
   'feedback.list-item-selected': {};
 };
 
 export type FeedbackEventKey = keyof FeedbackEventParameters;
 
 export const feedbackEventMap: Record<FeedbackEventKey, string | null> = {
-  'feedback.index-setup-button-clicked': 'Clicked Feedback Onboarding Setup Button',
-  'feedback.index-old-ui-clicked': 'Clicked Go To Old User Feedback Button',
```
Comment (ryan953) [inline static/app/utils/analytics/feedbackAnalyticsEvents.tsx:15]: moved these into the button itself with `analyticsEventKey` and `analyticsEventName`

File: static/app/views/feedback/feedbackListPage.tsx [hunk 4]
```diff
@@ -130,5 +106,6 @@ const Container = styled(FluidHeight)`
 `;
 
 const SetupContainer = styled('div')`
-  grid-column: 1 / 3;
+  overflow: hidden;
```
Comment (ryan953) [inline static/app/views/feedback/feedbackListPage.tsx:109]: this makes the setup message scroll when the window is really short.

**Review by michellewzhang — APPROVED** [review]
🧹

**Pull request merged**